### DISCUSSION TITLE
How to show secret detail?

### DISCUSSION THREAD

**Discussion by wcc526** [aquasecurity/trivy#8114] [Q&A]
### Question

How to show secret detail? when I use trivy `--format json -o x.json`, it only show the path of secret，I want the rule scan result.

```
 "Results": [
    {
      "Target": "/root/.ssh/id_rsa_deploy",
      "Class": "secret"
    }
 ]
```

### Target

Container Image

### Scanner

Secret

### Output Format

JSON

### Mode

Standalone

**Comment by knqyf263** [comment]
It means no issues are detected
